tests/script-change-testcase-return-type.php

```diff
@@ -4,5 +4,22 @@
     exit(0);
 }
 
-file_put_contents('tests/BaseTestCase.php', str_replace('public function setUp()', 'public function setUp(): void', file_get_contents('tests/BaseTestCase.php')));
-file_put_contents('tests/BaseTestCase.php', str_replace('public function tearDown()', 'public function tearDown(): void', file_get_contents('tests/BaseTestCase.php')));
+require_once './vendor/autoload.php';
+
+use Symfony\Component\Finder\Finder;
+
+$finder = new Finder();
+
+$finder->files()
+    ->ignoreVCS(false)
+    ->in(__DIR__)
+    ->name('*.php')
+    ->notName('script-change-testcase-return-type.php')->contains('use Orchestra\Testbench\TestCase;');
+
+foreach ($finder as $file) {
+    $absoluteFilePath = $file->getRealPath();
+    $fileNameWithExtension = $file->getRelativePathname();
```

Comment:
Redundant variable since you don't use it anywhere.

Reply:
Sure, forgot to remove them after testing